scoring/management/commands/import_plan_scores.py

```diff
@@ -16,14 +23,63 @@
 
 from django.conf import settings
 
+
+def download_github_release(org: str,
+                            repo: str,
+                            tag: str,
+                            dest: Path,
+                            private: bool = False):
+    """
+    Get and extract a release zip of a dataset from github.
+    If private is true, try to use PERSONAL_ACCESS_TOKEN
+    to access the repository. Should only be used as part of testing 
+    before a dataset is public.
+    """
+
+    file = f"https://github.com/{org}/{repo}/archive/refs/tags/{tag}.zip"
+
+    headers = None
+    if private:
+        token = os.environ.get("PERSONAL_ACCESS_TOKEN", None)
+        if token is None:
+            raise ValueError(
+                "Tried to access private repo, but no PERSONAL_ACCESS_TOKEN envkey.")
```

Comment:
I think this might need to be set in the environment bit of `docker-compose.yml` in order that it is set when the container is started.

Reply:
Added below, I think this means it will now work that adding a token from https://github.com/settings/tokens to .env will then be pulled through into the docker environment. 

The live version shouldn't need this because the files will be open, but a staging site might need someone to give it a temporary token?

Reply:
(Did some googling on this and annoyingly repo level tokens aren't available (despite codespaces clearly creating and using them behind the scenes).